projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -155,7 +155,7 @@ export function removeAllItemsFromCart() {
   cy.server();
   cy.route(
     'GET',
-    `${apiUrl}/rest/v2/electronics-spa/users/anonymous/carts/*?fields=*&lang=en&curr=USD`
+    `${apiUrl}/rest/v2/electronics-spa/users/*/carts/*?fields=*&lang=en&curr=USD`
```

Comment:
create a function because this is just copy pasting, so create a function that does

cy.route
addToCart()
cy.wait 

So we can just call that function in both of these places